solver/src/driver.rs

```diff
@@ -9,15 +9,19 @@ use anyhow::{Context, Result};
 use contracts::GPv2Settlement;
 use futures::future::join_all;
 use gas_estimation::GasPriceEstimating;
+use itertools::Itertools;
+use maplit::hashmap;
```

Comment:
What the difference between this and the standard library hash map?

Reply:
I'm guessing that the HashMap is the struct, and this maplit::hashmap is just a macro to create literal hashmaps?

Reply:
That's correct and in these cases I would suggest just checking the docs. From the use we see that the crate is called `maplit` so searching for that on crates.io or docs.rs shows the docs where it is explained.